config/rbac/role.yaml

```diff
@@ -6,6 +6,25 @@ metadata:
   creationTimestamp: null
   name: hypershfit-deployment-controller
```

Comment:
After I executed `make manifest`, the name became `manager-role`.

Reply:
oh, it is fixed [here](https://github.com/stolostron/hypershift-deployment-controller/pull/6/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R47)